presto-main/src/main/java/com/facebook/presto/execution/QueryManagerConfig.java

```diff
@@ -66,6 +69,8 @@
     private Duration queryMaxExecutionTime = new Duration(100, TimeUnit.DAYS);
     private Duration queryMaxCpuTime = new Duration(1_000_000_000, TimeUnit.DAYS);
 
+    private DataSize queryMaxScanRawInputBytes = DataSize.succinctDataSize(1, PETABYTE);
```

Comment:
Lets set the default to a higher number to avoid unexpected failures when people deploy this new version. Every sys admin should be able to set a reasonable value in their configurations. How about an exabyte (1000, PETABYTE)?

Reply:
sure, this field is adjustable anytime by DBAs. Default is revised to 1000PB.